examples/js-demo/babel.config.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
We shouldn't need to file if we provide [`alias`](https://v2.parceljs.org/features/module-resolution/#aliases) in `package.json`.

Reply:
Right! Done in https://github.com/algolia/react-recommendations/pull/6/commits/be611462b78ea8dc0603274a32af18d9b6964425